front/app/components/UI/Modal/index.tsx

```diff
@@ -93,15 +96,47 @@ const StyledCloseIconButton = styled(CloseIconButton)`
   `}
 `;
 
-const StyledFocusOn = styled(FocusOn)<{ width: number | string }>`
+// copy of the styled FocusOn container below
```

Comment:
I'm wondering whether we can avoid this duplication https://stackoverflow.com/questions/61519119/styled-components-two-different-elements-with-the-same-styles

But not sure it's worth it.

Reply:
Yeah, wanted to leave the other one completely untouched since it's what was there (and will be there) for the normal modal-based sign up/in flow. I think it's ok to duplicate a bit if it means leaving existing working code as-is

Reply:
Ok for me too.